.github/workflows/tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  python36:
+    name: Python3.6 tests
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - postgres: 10
+            python: 3.6
+            toxenv: py36
+            toxdir: psycopg3_c
+            implementation: c
+            # skip tests failing on importing psycopg3_c.pq on subprocess
+            # they only fail on Travis, work ok locally under tox too.
+            pytest_addopts: "-m 'not subprocess'"
+          - postgres: 11
+            python: 3.6
+            toxenv: py36
+            toxdir: psycopg3
+            implementation: python
+          - postgres: 11
+            python: 3.7
+            toxenv: py37
+            toxdir: psycopg3_c
+            implementation: c
+            pytest_addopts: "-m 'not subprocess'"
+          - postgres: 12
+            python: 3.7
+            toxenv: py37
+            toxdir: psycopg3
+            implementation: python
+          - postgres: 12
+            python: 3.8
+            toxenv: py38
+            toxdir: psycopg3_c
+            implementation: c
+            pytest_addopts: "-m 'not subprocess'"
+          - postgres: 12
+            python: 3.8
+            toxenv: py38
+            toxdir: psycopg3
+            implementation: python
+
+    env:
+      PGVER: ${{ matrix.postgres }}
+      PSYCOPG3_TEST_DSN: host=postgresql dbname=psycopg3_test
+      PSYCOPG3_IMPL: ${{ matrix.implementation }}
+      PYTEST_ADDOPTS: ${{ matrix.pytest_addopts }}
+      PGHOST: postgresql
```

Comment:
PGHOST was only needed by the CREATE DATABASE command. If we use a postgres docker image we can probably just use the PSYCOPG3_TEST_DSN env var.